host.yaml

```diff
@@ -163,3 +163,21 @@ zillion_options:
   # RetroArch doesn't make it easy to launch a game from the command line.
   # You have to know the path to the emulator core library on the user's computer.
   rom_start: "retroarch"
+adventure_options:
+  # File name of the standard NTSC Adventure rom.
+  # The licensed "The 80 Classic Games" CD-ROM contains this.
+  rom_file: "ADVNTURE.BIN"
+  # Set this to false to never autostart a rom (such as after patching)
+  # True for operating system default program for '.BIN'
+  # Alternatively, a path to a program to open the .bin file with (generally EmuHawk for multiworld)
+  rom_start: "EmuHawk.exe"
```

Comment:
default here is EmuHawk, but True in Utils.py, seems rather odd to have it diverge

Reply:
If there was a magic fixed path for EmuHawk, I'd just put that in both places, because an EmuHawk path is what I'd recommend there... but probably better to be consistent with True.  I'm also updating the comments there.